nbextensions/bulkImportCell/bulkImportCellStates.js

```diff
@@ -0,0 +1,47 @@
+define([], () => {
+    'use strict';
+    const states = [
+        {
+            state: {
+                mode: 'new'
+            },
+            ui: {
+                tab: {
+                    selected: 'configure',
+                    tabs: {
+                        configure: {
+                            enabled: true,
+                            visible: true,
+                        },
+                        viewConfigure: {
+                            enabled: false,
+                            visible: false
+                        },
+                        info: {
+                            enabled: true,
+                            visible: true
+                        },
+                        jobStatus: {
+                            enabled: false,
+                            visible: true
+                        },
+                        results: {
+                            enabled: false,
+                            visible: true
+                        },
+                        error: {
+                            enabled: false,
+                            visible: false
+                        }
```

Comment:
is this tab still going to be used?

Reply:
Which one, the error tab? I think that was the plan? I might be mistaken.

Reply:
Yes -- I was just looking through the mockups to see in what circumstances it would appear, but asking design would probably be quicker!